code/WorkInProgress/nadir_antenna.dm

```diff
@@ -408,7 +437,7 @@ var/global/obj/machinery/communications_dish/transception/transception_array
 				src.is_transceiving = FALSE
 
 
-	proc/receive_a_thing(var/netnumber,var/atom/movable/thing2get)
+	proc/receive_a_thing(var/netnumber,var/atom/movable/thing2get,var/was_manual = FALSE)
```

Comment:
```suggestion
	proc/receive_a_thing(var/netnumber,var/atom/movable/thing2get, var/was_manual = FALSE)
```

Reply:
There are no spaces separating the proc arguments in general; would you like spaces of this sort put into all the procs? It'd be weird to do just one.

Reply:
https://hackmd.io/@goonstation/docs/%2F%40goonstation%2Fcode#Whitespace